### PR TITLE
Feature/model label zip

### PR DESCRIPTION
**Pull request by Perryongwq**
# Description

> Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Create a container in the menu page for zipping of the model (.keras) and label (.txt) together files and downloading of the zipped files. 

## Fixes / issues

1. Create backend program for zipping of the model and label files
2. Create a new container for hyperparameters and modelzip/save components
3. Create a frontend program to enable the function of modelzip
4. Clean up the backend code and improve UI/UX on alert notification 

## Type of change

> Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: frontend/src/pages/CMT/containers/MenuChildren/components/ModelSave.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import zipModel from "../../../utils/zipModel";
+import Swal from "sweetalert2";
+
+import DropBox from "../../../../../containers/common/DropBox";
+import Button from "../../../../../containers/common/Button";
+
+const ModelSave = ({ refresh }) => {
+  const { drop, setDrop, parameters, setParameters } = useContext(AppContext);
+
+  const set_parameters = (e) => {
```
Comment (cljiahao) [inline frontend/src/pages/CMT/containers/MenuChildren/components/ModelSave.js:13]: setDrop should use another naming. If not the dropbox in hyperparameter and model save are the same.

File: frontend/src/pages/CMT/containers/MenuChildren/components/ModelSave.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        confirmButtonText: "OK",
+      });
+      return;
+    }
+
+    const modelName = drop.model.selected.replace(/^temp\//, "");
+
+    const blob = await zipModel(modelName);
+    const url = URL.createObjectURL(blob);
+    const link = document.createElement("a");
```
Comment (cljiahao) [inline frontend/src/pages/CMT/containers/MenuChildren/components/ModelSave.js:39]: Please use react elements instead.

File: backend/src/apis/routes/CMT_zip_file.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import os
+import zipfile
+from fastapi import FastAPI, APIRouter, HTTPException
+from fastapi.responses import FileResponse
+from pydantic import BaseModel
+
+
+router = APIRouter()
+
+class Directory:
```
Comment (cljiahao) [inline backend/src/apis/routes/CMT_zip_file.py:10]: There is already a Directory file. Please use that instead.

File: backend/src/apis/routes/CMT_zip_file.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+class ModelName(BaseModel):
+    modelname: str
+
+dire = Directory.models_path
+temp_folder = 'temp'
+
+@router.post("/zip_model/")
+async def zip_model(model: ModelName):
+    model_base_path = os.path.join(dire, temp_folder, model.modelname)
```
Comment (cljiahao) [inline backend/src/apis/routes/CMT_zip_file.py:27]: temp_folder can be made in the directory class

**Review by cljiahao — CHANGES_REQUESTED** [review]
